tests/test_102_NumFieldsSelect_01.phpt

```diff
@@ -3,7 +3,6 @@ IBM-DB2: db2_num_fields() - select
 --SKIPIF--
 <?php
   require_once('skipif.inc');
-  require_once('skipif3.inc');
```

Comment:
This change seems unrelated to all the other changes.

Reply:
Actually not - it makes the LUW version of this test run on i, since skipif3 handles if a test runs on i (PASE or Db2 Connect). It's incredibly obviously named, no?

I'm guessing like v5r4/6.1 didn't support VALUES...

Reply:
Gotcha. From the diff it looks like an outlier since all the other changes were deleting V6 variants of the tests.

Reply:
Yeah, anything that was TBD removed was prob me restoring the non-PASE version fo the test to work on all platforms...